db.js

```diff
@@ -0,0 +1,51 @@
+const knex = require('knex');
+const knexConfig = require('./knexfile.js');
```

Comment:
This file should probably be named something more descriptive, like dishesModel.js, or something like that